Automattic/wp-calypso
# Jetpack Manage: Table Atelier - Revisit the Onboarding Tour

Related to https://github.com/Automattic/jetpack-manage/issues/325

## Proposed Changes

* This PR fixes a problem with the routing for dashboard tours, the tours param was not being persisted then updating the URL
* This PR updates the dashboard walkthrough tour for the sites dashboard v2
* This PR also updates the link for the "Add new site" tour.

## Testing Instructions

* The first thing to test is if our current dashboard still works, you can do so by following the instructions found in https://github.com/Automattic/wp-calypso/pull/85440
* Then enable the new Sites Dashboard using the feature flag:
   * Feature Flag: jetpack/manage-sites-v2-menu To activate the Sites V2 menu. You can deactivate the feature flag using this query in the URL: ?flags=-jetpack/manage-sites-v2-menu
* Test the tour in the new Sites Dashboard
* With the feature flag enabled go to the overview page and click over `Learn how to add new sites` and check if you get directed to the new sites dashboard.

Note: the `Add new sites` tour has 2 steps, the second step is triggered after a new Jetpack site is successfully connected after step 1, but the user will get redirected to a hardcoded URL located in  https://github.com/Automattic/wp-calypso/blob/3a922b9ef1d0ef3d18a02497ac6bfefa9e808b48/client/jetpack-connect/constants.js#L29 when we go live this URL has to be updated to the new sites url.

## Pre-merge Checklist
